.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: go
 go:
   - 1.6
```

Comment:
@piosz Just want to add go 1.7 since other kubernetes repo seems to migrate to it. I have no favor on testing agaginst go 1.6. :)